src/features/pages/home/live-pricing/use-live-columns.tsx

```diff
@@ -0,0 +1,140 @@
+                        <Typography.Paragraph
+                            weight="bold"
+                            font_family="UBUNTU"
+                            size={is_mobile ? 'small' : 'medium'}
+                        >
+                            <Localize translate_text="_t_Bid price_t_" />
+                        </Typography.Paragraph>
+                    </Flex.Box>
+                ),
+                cell: (info) => (
+                    <Flex.Box className={table_header_cell}>
+                        <Typography.Paragraph size={is_mobile ? 'small' : 'medium'}>
+                            {info.getValue()}
+                        </Typography.Paragraph>
+                    </Flex.Box>
+                ),
+            }),
+            liveMarketColumnHelper.accessor('ask', {
+                header: () => (
+                    <Flex.Box className={table_header_cell}>
+                        <Typography.Paragraph
+                            weight="bold"
+                            font_family="UBUNTU"
+                            size={is_mobile ? 'small' : 'medium'}
+                        >
+                            <Localize translate_text="_t_Ask price_t_" />
+                        </Typography.Paragraph>
+                    </Flex.Box>
+                ),
+                cell: (info) => (
+                    <Flex.Box className={table_header_cell}>
+                        <Typography.Paragraph size={is_mobile ? 'small' : 'medium'}>
+                            {info.getValue()}
+                        </Typography.Paragraph>
+                    </Flex.Box>
+                ),
+            }),
+            liveMarketColumnHelper.accessor('spread', {
+                header: () => (
+                    <Flex.Box className={table_header_cell}>
+                        <Typography.Paragraph
+                            weight="bold"
+                            font_family="UBUNTU"
+                            size={is_mobile ? 'small' : 'medium'}
+                        >
+                            <Localize translate_text="_t_Spread_t_" />
+                        </Typography.Paragraph>
+                    </Flex.Box>
+                ),
+                cell: (info) => (
+                    <Flex.Box className={table_header_cell}>
+                        <Typography.Paragraph size={is_mobile ? 'small' : 'medium'}>
+                            {info.getValue()}
+                        </Typography.Paragraph>
+                    </Flex.Box>
+                ),
+            }),
+            liveMarketColumnHelper.accessor('day_percentage_change', {
+                header: () => (
+                    <Flex.Box className={table_header_cell}>
+                        <Typography.Paragraph
+                            weight="bold"
+                            font_family="UBUNTU"
+                            size={is_mobile ? 'small' : 'medium'}
```

Comment:
```suggestion
                            size={is_mobile && 'small' }
```

Reply:
This throws TS error `Type '"" | "small"' is not assignable to type 'TTextSize'.`